### ISSUE TITLE
Upgrade broadway version

### ISSUE THREAD

**Issue by AlessandroMinoccheri** [nWidart/Laravel-broadway#35]
Is possible to use broadway 2.x instead of broadway ~1.0.0?
It could be a new version to be update with broadway library, what do you think?

**Comment by Lavoaster** [comment]
Yeah, that is my intention to do at some point, however, I haven't touched the project that uses this in quite some time and I'm hesitant to push stuff up here prior to testing it out there first.

**Comment by AlessandroMinoccheri** [comment]
What is necessary to do for you? Maybe I can help if is possible
